task/oci-cli/0.1/oci-cli-task.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: oci-cli-task
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.54.0"
+    tekton.dev/categories: CLI
+    tekton.dev/tags: cli
+    tekton.dev/displayName: "oracle cli task"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  params:
+    - name: tenancy_ocid
+      description: "The OCID of the tenancy"
+    - name: user_ocid
+      description: "The OCID of the user"
+    - name: fingerprint
+      description: "The fingerprint of the API key"
+    - name: private_key
+      description: "The private key content (base64 encoded)"
```

Comment:
should we mount these values via kubernetes secrets?

Reply:
@vinamra28 done i have added tag .. and passed sensitive data as secret

Reply:
@vinamra28  ?

Reply:
@vinamra28 any update ?

Reply:
@vinamra28  is this project archived ?? no response for two weeks .. hence asking

Reply:
hi @arunvel1988, apologies for getting back late on this. I am not part of another project so didn't get time to review the PRs here. I'll try to be more active from now on.